learn/guides/ballerina-openapi-support.md

```diff
@@ -57,26 +59,25 @@ The service generation process is complete. The following files were created.
 -- hello_service.bal
 -- types.bal
 ```
-### Generate a Ballerina service from tags
+### Generate from tags
 To generate the Ballerina service stub with a subset of tags defined in an OpenAPI contract, use the `--tags` option and specify the tags you need as specified in the OpenAPI definition.
 ```bash
 $ bal openapi -i <openapi-contract> [--tags <"tag1","tag2">]
 ```
 
-**Example:**
+For example, 
+
 ```bash
 $ bal openapi -i hello.yaml --tags "pets", "list"
 ```
 Once you execute the command, only the operations related to the given tags will be included in the generated service file.
 
->**Info:** For more command options, see
-[OpenAPI to Ballerina command reference](#openapi-to-ballerina-command-reference)
+>**Info:** For more command options, see [Command reference](#command-reference).
 
-## Export an OpenAPI contract from a Ballerina service (code-first approach)
+## Export an OpenAPI contract
 
-You can convert your Ballerina service APIs into human-readable or machine-readable documents such as OpenAPI documents by using the Ballerina to OpenAPI command as follows.
+Exporting an OpenAPI contract from a Ballerina service is named as the `code-first approach`. You can convert your Ballerina service APIs into human-readable or machine-readable documents such as OpenAPI documents by using the Ballerina to OpenAPI CLI Tool as follows.
```

Comment:
Following the `API code-first approach`, you can convert your Ballerina service APIs into human-readable or machine-readable documents such as OpenAPI documents by using the Ballerina to OpenAPI CLI Tool as follows.

Reply:
```suggestion
Following the `API code-first approach`, you can convert your Ballerina service APIs into human-readable or machine-readable documents such as OpenAPI documents by using the Ballerina to OpenAPI CLI Tool as follows.
```